scripts/manager_char_4e_tmt.lua

```diff
@@ -0,0 +1,74 @@
+-- 
+-- Please see the license.txt file included with this distribution for 
+-- attribution and copyright information.
+--
+
+rPowerContainsDataCondition = nil;
+rPowerUsedEvent = {
+    sName = "power_used_event",
+    aParameters = { 
+        "rSource",
+        "rPower"
+    };
+};
+
+function onInit()
+    TriggerManager.defineEvent(rPowerUsedEvent);
+    initializeConditions();
+end
+
+function initializeConditions()
+    rPowerContainsDataCondition = {
+        sName = "power_used_contains_data",
+        fCondition  = powerContainsDataCondition,
+        aRequiredParameters = {
+            "rSource",
+            "rPower"
+        },
+        aConfigurableParameters = {
+            {
+                sName = "sPowerProperty",
+                sDisplay = "power_used_property_parameter",
+                sType = "combo",
+                aDefinedValues = {
+                    "power_used_property_name",
+                    "power_used_property_keywords",
+                    "power_used_property_recharge",
+                    "power_used_property_action",
+                    "power_used_property_range",
+                }
+            },
+            {
+                sName = "sPowerContains",
+                sDisplay = "power_used_contains_parameter",
+                sType = "string"
+            }
+        }
+    }
+
+    TriggerManager.defineCondition(rPowerContainsDataCondition);
+end
+
+function powerContainsDataCondition(rTriggerData, rEventData)
+    -- If contain property is empty, return true
```

Comment:
If I'm understanding correctly, I don't think there's a real use case for an empty sPowerContains parameter, other than user error/oversight. In that event I think have the trigger not work would alert them to that more quickly?

Reply:
I can change it to that. The idea was if someone leaves it empty I don't want it throwing errors, but I'm fine either way.

Reply:
I leave it entirely to your judgement.